internal/grpctest/grpctest.go

```diff
@@ -48,16 +49,22 @@ type Tester struct{}
 // Setup updates the tlogger.
 func (Tester) Setup(t *testing.T) {
 	TLogger.Update(t)
+	// TODO: There is one final leak around closing connections without completely
+	//  draining the recvBuffer that has yet to be resolved. All other leaks have been
+	//  completely addressed, and this can be turned back on as soon as this issue is
+	//  fixed.
+	leakcheck.SetTrackingBufferPool(errorer{t: t})
 }
 
 // Teardown performs a leak check.
 func (Tester) Teardown(t *testing.T) {
+	leakcheck.CheckTrackingBufferPool()
```

Comment:
Does anything change in terms of tests passing/failing, if we perform this check after the check for goroutines? Or was there a specific reason to perform this check first? If so, could we have a comment for that. Thanks.

Reply:
Probably not? I can see if it does